storages/python/python-storage-plugins/yaml.py

```diff
@@ -89,7 +93,8 @@ def save(self, inst: dlite.Instance):
 
         """
         self._data[inst.uuid] = inst.asdict(
-            soft7=dlite.asbool(self.options.soft7)
+            soft7=dlite.asbool(self.options.soft7),
+            uuid=self.single,
```

Comment:
Not sure what this means. But I'm guessing it means that only a single instance from `inst` is returned? Or multiple?

Reply:
It means that it is only when we we to a yaml file in single-entity form that we want to include the UUID. In multiple-entity form, the UUID (or possible the URI) will already be provided as the key for the entity.